include/z64actor.h

```diff
@@ -187,8 +187,8 @@ typedef struct ActorShape {
 //
 #define ACTOR_FLAG_23 (1 << 23)
 
-//
-#define ACTOR_FLAG_24 (1 << 24)
+// When colliding with Player's body AC collider, a "thump" sound will play indicating his body has been hit
+#define ACTOR_FLAG_HIT_PLAYER_THUMP_SFX (1 << 24)
```

Comment:
the sfx is called "BODY_HIT" instead of "THUMP", maybe something like `ACTOR_FLAG_PLAY_BODY_HIT_SFX`?

Reply:
This is the exact name I had years ago. There was at least one comment that didnt like it, at least initially https://github.com/zeldaret/oot/pull/1122#discussion_r794669674

I think beyond the name of the sound, the phrasing I have currently is trying to make it clear that the sfx plays when the player is hit. While `PLAY_BODY_HIT_SFX` is just saying "play the hit sound", but when?
I guess context makes it clear when it will happen, but idk

Reply:
ACTOR_FLAG_PLAY_SFX_FOR_BODY_HIT?

I think "body" is good, if I read something like "the punch was a body hit" in an article I would assume it means the chest or stomach area, and this "thump" sounds like that to me. But if it hit his arm or something the thump sound would not be appropriate

Reply:
That could work. Another piece thats missing is "player". But the flag name will start to become word soup (like my original suggestion kind of already is)
Im fine with leaving it out/leaving it for the comment to explain. Just want to note it.

Reply:
good point, I'd be fine with `true `ACTOR_FLAG_PLAY_SFX_FOR_PLAYER_BODY_HIT` too but yeah it is rather long now

Reply:
is "play" super necessary?
`ACTOR_FLAG_SFX_FOR_PLAYER_BODY_HIT`
`ACTOR_FLAG_SFX_ON_PLAYER_BODY_HIT`
?

Reply:
those work for me (slightly preference for FOR rather than ON)